packages/react-liveness/src/components/FaceLivenessDetector/shared/Toast.tsx

```diff
@@ -1,28 +1,46 @@
 import * as React from 'react';
-import { Flex, View, ViewProps } from '@aws-amplify/ui-react';
+import { Flex, View, ViewProps, useTheme } from '@aws-amplify/ui-react';
 
 import { LivenessClassNames } from '../types/classNames';
 
 interface ToastProps extends ViewProps {
   variation?: 'default' | 'primary' | 'error';
   size?: 'medium' | 'large';
   heading?: string;
+  isInitial?: boolean;
 }
 
+const toastStyle = {
+  borderRadius: 'unset',
+  fontSize: 'xxl',
+  padding: '0 var(--amplify-space-xs)',
+};
+
+const maxWidthStyle = { base: '100%' };
```

Comment:
Can we move all this stuff to CSS and classnames? It is kind of hard to understand what is happening